extension/types/hiraganaLookupMap.ts

```diff
@@ -0,0 +1,247 @@
+export const kanaToHiraganaNormalizationMap: Record<string, string> = {
```

Comment:
I wouldn't consider this a type at all since it contains implementation details. But it's definitely data related so to me it's easier to group stuff when thinking about the product rather than the language features.

Reply:
Types folder has been removed, constants have been added to the lookup map and the file has been renamed to `character_info.ts`